src/app/tabs/camera.cpp

```diff
@@ -0,0 +1,143 @@
+#include <app/config.hpp>
+#include <app/tabs/camera.hpp>
+#include <app/window.hpp>
+#include <app/theme.hpp>
+
+CameraTab::CameraTab(QWidget *parent) : QWidget(parent)
+{
+    QStackedLayout *stack = new QStackedLayout(this);
+    stack->addWidget(connect_widget());
+    stack->addWidget(cam_widget());
+
+    connect(this, &CameraTab::stream_connected, this, [stack]() {stack->setCurrentIndex(1);} );
+    connect(this, &CameraTab::stream_disconnected, this, [stack]() {stack->setCurrentIndex(0);} );
+
+    player = NULL;
+    // connect_stream(); TODO: offer auto-connect option

```

Comment:
should the auto connection be handled here? Wouldnt it be easier to just check the config after the camera tab is initialized and call connect then